corehq/apps/cleanup/management/commands/populate_sql_model_from_couch_model.py

```diff
@@ -194,11 +208,24 @@ def add_arguments(self, parser):
                 models that don't support verification.
             """,
         )
+        parser.add_argument(
+            '--fixup-diffs',
+            action='store_true',
+            dest='fixup_diffs',
+            default=False,
+            help="Update rows in SQL that do not match their corresponding Couch doc.",
```

Comment:
Taking updates off of the default code path resolves a race condition:

- normal HQ operation loads a Couch doc
- a `PopulateSQLCommand` migration loads the same doc in a batch of 100 docs
- HQ operation modifies and saves the Couch doc, which also syncs changes to SQL (the migration's copy of the doc is now stale)
- migration calls `update_or_create_sql_object` which overwrites above changes, reverting SQL to the state of its stale Couch doc

Assuming the model sync logic is working correctly, all diffs reported by the migration should be transient, and should eventually be resolved by non-migration code if the migration only does inserts (no updates).

The `--fixup-diffs` option is provided so the migration command can be used to resolve bugs in the Couch to SQL sync logic that cause persistent differences between Couch and SQL. It is inefficient because each updated doc must be saved individually. Therefore, if a bug is discovered and most documents have already been migrated but PR 2 has not yet been merged, it may be more efficient to delete all SQL rows (since Couch is still the source of truth), fix the bug, and redo the migration.